oauth-function-template/handlers/authorizationCode.js

```diff
@@ -35,12 +36,16 @@ const refreshHandler = (req, res, ctx) => {
             }
             return res.status(500).send(packageError(result));
           }
-          const url = provider.innerAuthorizationUrl;
+          const url = (provider.innerAuthorization && provider.innerAuthorization.url) || provider.innerAuthorizationUrl;
```

Comment:
if we already fetched `lodash` to this module then let's use it here instead of` &&`